thunder/torch/__init__.py

```diff
@@ -4070,7 +4070,14 @@ def sigmoid(a: TensorLike, /) -> TensorLike:
 
 # CompositeImplicitAutograd - don't register decomp
 @torchsymbol(torch.softmax, torch.nn.functional.softmax, is_method=True)
-def softmax(a: TensorLike, /, dim: int, *, dtype: None | dtypeLike = None) -> TensorLike:
+def softmax(
```

Comment:
Instead of modifying the symbol, what about creating a non-symbol function that calls the symbol, like this:

```
@torchsymbol(torch.softmax, torch.nn.functional.softmax, is_method=True)
def _softmax(...):
  # The symbol doesn't take _stacklevel, so things don't need to reason about it
  ...

def softmax(..., _stacklevel:int = 3):
  # Doesn't pass _stacklevel, just ignores it
  return _softmax(...)
```

Would something like that work?

Reply:
[`torch.softmax`](https://pytorch.org/docs/stable/generated/torch.softmax.html#torch-softmax) doesn't have `_stacklevel` as one of its arguments but [`torch.nn.functional.softmax`](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html#torch.nn.functional.softmax) does while the former is said to be an alias for the latter. I'll take your suggestion.